src/interface/desktop/chat.html

```diff
@@ -888,10 +888,10 @@
         }
 
         function renderFirstRunSetupMessage() {
-            first_run_message = `Hi 👋🏾, to get started:
+            first_run_message = `<p class="inline-chat-heading">Hi 👋🏾, to get started:<p>
             <ol>
-                <li>Generate an API token in the <a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToWebSettings()">Khoj Web settings</a></li>
-                <li>Paste it into the API Key field in the <a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToSettings()">Khoj Desktop settings</a></li>
+                <li class="inline-chat-text">Generate an API token in the <a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToWebSettings()">Khoj Web settings</a></li>
+                <li class="inline-chat-text">Paste it into the API Key field in the <a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToSettings()">Khoj Desktop settings</a></li>
```

Comment:
Let's make this text a bit more intuitive.

```suggestion
                <li class="inline-chat-text">Generate an API token<a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToWebSettings()">Khoj Web settings</a></li>
                <li class="inline-chat-text">Paste it into the API Key field<a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToSettings()">Khoj Desktop settings</a></li>
```